ui/raidboss/data/04-sb/raid/o4n.txt

```diff
@@ -0,0 +1,98 @@
+# DELTASCAPE V4.0--O4N
+# EXDEATH
+
+# Enrage appears to be 25 minutes. No way to accurately warn about it
+# as the second phase is HP-dependent.
+
+hideall "--Reset--"
+hideall "--sync--"
+
+# -ii 24BA 24BC 24BE 24C3 24C5
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+0 "Start"
+2.5 "--sync--" sync /:Exdeath:366:/ window 2.5,1
+9.5 "Doom" sync /:Exdeath:24B7:/ window 9.5,5
+19.1 "Blizzard/Fire/Thunder III" sync /:Exdeath:24B[9BD]:/
```

Comment:
Is this really just 7 random abilities in a row?

Reply:
More that it's two different triple choices. First the non-empowered stuff, then the empowered stuff. Looking back through my logs, I can't find an example of not-Thunder second, so I'll make that change. However, the empowered set is definitely random.

Reply:
Hmm. And now I'm not so sure. I know for a fact I've seen random empowered sets, but I can't find any in my logs. I wonder whether that was a confusion from Stormblood, or whether it changed.